### DISCUSSION TITLE
Incompatible Layers

### DISCUSSION THREAD

**Discussion by kashifmahi** [nftchef/art-engine#127] [Help + Q&A]
I want to use the incompatible layers function, but it's not working 

For example, in the Background layer, there is a red Background trait and I want it that I would never combine with the red Hat trait from the Hats layer, and also it never pickup the red colored character from the character traits folder
How do I change this?

const incompatible = { Red: ["Dark Long"], "Name with spaces": ["buzz","rare-Pink-Pompadour" ] // directory incompatible with directory example White: ["rare-Pink-Pompadour"], };

**Comment by cryptofannatic** [comment]
did you figure this out @kashifmahi

**Comment by BakeSloth** [comment]
const incompatible = { 
"red Background trait": ["red colored character","red Hat trait"]
};

Maybe this can help? https://github.com/nftchef/art-engine/discussions/150
